docs/news/2020-07-06-Dockstore-1.9.0.rst

```diff
@@ -0,0 +1,39 @@
+Dockstore 1.9.0
+===============
+
+Highlighted new features include:
+---------------------------------
+
+-  Language plugin support, allowing for additional support of Galaxy workflows as a preview feature
+-  Performance improvements for a large variety of endpoints
+-  GitHub app support in preview to allow for automated update of workflow content
+-  Implementation of the `TRS v2 final standard <https://github.com/ga4gh/tool-registry-service-schemas/releases/tag/2.0.0>`_
+
+   -  TRS v2 beta standard support is deprecated, but still present
+
+-  Capture of file and docker image checksums from GitHub, quay.io, Gitlab, and Docker Hub on workflow snapshot to support immutable workflows (see breakdown for details here_) support for linking ORCID profiles user profiles
+
+   -  Currently displayed in organization and stargazer views
+
+-  A large number of usability improvements and fixes to the user interface website
+-  Partial API migration from swagger 2.0 to openapi 3.0 for the Dockstore API
+-  A large variety of security updates and bug fixes
+
+As always, see our full list of changes on `GitHub`_
+
+Breaking changes
+----------------
+
+Major
+~~~~~
+
+None intended
+
+Minor
+~~~~~
+
+- If upgrading the Dockstore CLI, our Dockstore script has changed and should be downloaded anew from the onboarding wizard
+- CLI website management features (refresh) will be broken until CLI version is upgraded, workflow launching should remain
```

Comment:
I feel this is phrased slightly oddly, talking about website management features, and not sure what "workflow launching should remain" means (it will continue to work?). I would just say something like:

The CLI `refresh` command only works in 1.9 (needs work too).